gdpr/tests/test_gdpr_views.py

```diff
@@ -0,0 +1,173 @@
+import pytest
+from helusers.authz import UserAuthorization
+from rest_framework import status
+from rest_framework.test import APIRequestFactory, force_authenticate
+
+from gdpr.views import MvjGDPRAPIView
+from plotsearch.enums import InformationCheckName
+
+
+def _find_matching_dicts(data: list, key: str, match_value: str) -> list:
+    """
+    Recursively find all dictionaries in a nested structure that match the key-value pair.
+    """
+    matches = []
+    if isinstance(data, dict):
+        if data.get(key) == match_value:
+            matches.append(data)
+        for value in data.values():
+            matches.extend(_find_matching_dicts(value, key, match_value))
+    elif isinstance(data, list):
+        for item in data:
+            matches.extend(_find_matching_dicts(item, key, match_value))
+    return matches
+
+
+@pytest.mark.django_db
+def test_api_get_user_data(
+    settings,
+    user_factory,  # users
+    area_search_factory,  # plotsearch
+    area_search_attachment_factory,  # plotsearch
+):
+    settings.GDPR_API_QUERY_SCOPE = "gdprquery"
+    settings.OIDC_API_TOKEN_AUTH = {
+        "API_AUTHORIZATION_FIELD": "authorization.permissions.scopes",
+    }
+
+    user = user_factory(first_name="Jari", last_name="Kuningas")
```

Comment:
This is a real person. I would use something like "UserFirstName" and "UserLastName"

Reply:
... unless it's required for the Suomi.fi authentication

Reply:
Not required, fair point, although it is not a real name

Reply:
I mean, that person exists in the real world, I googled it :) But not in our system.